content/posts/Interactive-book-new-avatar.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Interactive Book's New Avatar!"
+date: 2021-05-26T11:12:38+05:30
+draft: false
+author: Shivansh Srivastava
+type: post
+url: "/posts/interactive-book-revamp/"
+---
+
+
+We are excited 🎊 to launch the all new Interactive Book, completely filled with content related to digital logic design! The idea behind the revamp was to create new and interesting content along with enhancing the existing features of the interactive book. This will for sure benefit the students, professors and learners across the globe. We hope you experience enriched learning!
+
+![Interactive Book Launch](/images/Banner-IB-Launch.png)
+
+## What is [Interactive Book](https://learn.circuitverse.org)?
+
+The aim of the project is to create an online interactive guide for digital logic design. The primary goal is to develop an open source book with quality content that teaches digital logic design. It will enable students to learn digital design by interacting with circuits, truth table, and other interactive elements as they proceed through the book. The professors and students all over the world can read and contribute to the same.
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+## What's popping? 🤔
+
+- Rearranged current content intro new tree structure
+- Created content which is not covered in the earlier version. You will get to learn more from the enhanced content along with detailed diagrams.
+- Included Logic Circuits designed with Circuitverse for all the topics for a better understanding! 
+- Enhanced content
+- Pop Quizzes for each Topic
+- Detailed Contributing Guidelines
+- The book is now licensed under the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) and copyrighted by contributors to [CircuitVerse](https://circuitverse.org/)
+
+Want to know more? View the final report of the project [here](https://blog.circuitverse.org/posts/gsod2020-cvib-final-report-dvls/).
+
+## Get to know the brilliant minds behind the curtains!🥁
+
+I was offered an opportunity by [Satvik](https://github.com/satu0king) and [Aboobacker](https://github.com/tachyons) to mentor the projects listed by Circuitverse for Google Season Of Docs 2020, and after carefully reviewing all the proposals we chose [Daniel Vicente Lühr Sierra](https://github.com/danielluehr) for **"CircuitVerse Interactive Book Consolidation and Improvements"** Project.
+
+Daniel completely rebuilt the website, from updating the current structure of the Interactive Book, adding circuits and diagrams, to finally creating new matter and improvising the previous ideas.
```

Comment:
A line introducing Daniel and mentioning his experience would be a great addition